database/tbcd/level/level.go

```diff
@@ -1736,7 +1736,7 @@ func (l *ldb) BlockUtxoUpdate(ctx context.Context, direction int, utxos map[tbcd
 	log.Tracef("BlockUtxoUpdate")
 	defer log.Tracef("BlockUtxoUpdate exit")
 
-	if !(direction == 1 || direction == -1) {
+	if direction != 1 && direction != -1 {
```

Comment:
I am a hard no on these changes.

Reply:
This change wasn't intentional - I missed this when reviewing the automated changes from the new staticcheck rules. Sorry.